### ISSUE TITLE
Specifying separate priors for each SNP in multidog

### ISSUE THREAD

**Issue by gtbil** [dcgerard/updog#27]
Thanks for making an awesome package!

Is there currently a way to specify a different prior for each SNP in `multidog`? I didn't see it in the documentation, but wanted to ask before I attempted to hack a way around it.

The use case I'm thinking of is genotyping in allotetraploids. I expect all SNPs to only have three genotype classes, and to segregate as in a diploid, but some of them will look sort of like ploidy = 4 (with three genotype classes like AAAA, AAAB, and AABB present and two genotype classes ABBB and BBBB not present) and some will look like ploidy = 2 (with all three genotype classes observed).

The work-around idea I had was to split my SNPs into separate pieces based on which prior and/or ploidy they need (using some heuristics) and run `multidog` separately for each, then merge them.

Thanks!

- Grant

**Comment by dcgerard** [comment]
Hey @gtbil,

Thanks for trying out `{updog}`!

Right now, this isn't available. But it would not be hard to implement, so I could do that when I get a chance in the next couple months.  But are you saying that you would like the `ploidy` argument the vary as well from SNP to SNP?

Best,
David

**Comment by gtbil** [comment]
Yes! Exactly, it would be great to have the ability to vary the `ploidy` argument and the `prior_vec` argument by SNP. I can provide test data for some example SNPs (with my manual genotype calls) I have in hand if it would be helpful for testing, or I can test myself. 

Thanks a lot,
Grant
